pages/osx/brew.md

```diff
@@ -14,14 +14,18 @@
 
 `brew list`
 
-- Update an installed formula (passing no formula updates all installed formulae):
+- Update an installed formula (passing no `{{formula}}` updates all installed formulae):
 
 `brew upgrade {{formula}}`
 
-- Update brew itself:
+- Fetch the newest version of Homebrew and all formulae from GitHub:
 
 `brew update`
 
+- Show formulae that have an updated version available:
```

Comment:
I think `updated` is better suited here. `new` could be used to signify new/novel formulae.